sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiDynamicDestinationsBeamRow.java

```diff
@@ -86,11 +86,11 @@ public DescriptorProtos.DescriptorProto getDescriptor(boolean includeCdcColumns)
     public StorageApiWritePayload toMessage(
         T element, @Nullable RowMutationInformation rowMutationInformation) throws Exception {
       String changeType = null;
-      long changeSequenceNum = -1;
+      String changeSequenceNum = null;
```

Comment:
if `rowMutationInformation` is null (as it is in most use-cases), what change sequence value do we set in the DynamicMessage? Can we set a null value?

Same question for the similar changes in Avro and TableRow paths